requirements.txt

```diff
@@ -0,0 +1,8 @@
+cycler==0.10.0
+env==0.1.0
+kiwisolver==1.2.0
+matplotlib==3.2.1
+numpy==1.18.4
+pyparsing==2.4.7
+python-dateutil==2.8.1
+six==1.14.0
```

Comment:
six? Where is this used?

Reply:
Six is a basic venv depency package. It is used to smooth out the differencies of a python 2.X interpereter vs a python 3.X interpreter